packages/cinterop/src/darwin/kotlin/io/realm/internal/interop/RealmInterop.kt

```diff
@@ -1352,6 +1353,35 @@ actual object RealmInterop {
         return nativePointerOrNull(currentUserPtr)
     }
 
+    actual fun realm_app_get_all_users(app: RealmAppPointer): List<RealmUserPointer> {
+        memScoped {
+            // Discover number of users
```

Comment:
I would add to the comment that this is needed to create the array of users using the correct allocation size. It was tempting for me to think that calling `realm_app_get_all_users` twice is unnecessary (until I read the docs in `realm.h` 🐒)